lib/styles/spacings.dart

```diff
@@ -0,0 +1,17 @@
+class CocktailAppSpacings
+{
+  // examples
```

Comment:
Remove this comment if it not needed anymore

Reply:
Suggested changes already pushed. Thank you for your comments and help @keylinwu-dnamic !